### DISCUSSION TITLE
Welcome to AL-Go Discussions!

### DISCUSSION THREAD

**Discussion by freddydk** [microsoft/AL-Go#105] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by freddydk** [comment]
I have added the currently known list of ideas to the discussions board.
Feel free to join the discussion on existing ideas, share your views or ideas as well.

**Comment by curateideas** [comment]
Hi, I'm Christian from Germany. I work as a development facilitator for a Microsoft Partner. We're helping our project teams to use efficient tools for their projects to deliver value to our #MSDYN365BC  customers. We use this community to share knowledge with all the members to learn and teach.

**Comment by DanielMagMat** [comment]
Hi, I'm Daniel from Mexico. I am a Reselling and Cloud Solutions Microsoft Partner focusing on Business Applications. We see in Business Central a good match for our SMB customers. We are eager to drive value within the community for partners and customers. We thank in advance for the creation of this community, so we can share and work as an extended team.
